snapcraft/cli/store.py

```diff
@@ -341,19 +341,34 @@ def export_login(login_file: str, snaps: str, channels: str, acls: str,
     # This is sensitive-- it should only be accessible by the owner
     private_open = functools.partial(os.open, mode=0o600)
 
-    # mypy doesn't have the opener arg in its stub. Ignore its warning
-    with open(login_file, 'w', opener=private_open) as f:  # type: ignore
-        store.conf.save(config_fd=f)
+    # Support a login_file of '-', which indicates a desire to print to stdout
```

Comment:
Mind adding the `-` to the login command documentation?

Reply:
Done.